docs/best-practices/continuous-integration/codemagic.md

```diff
@@ -0,0 +1,28 @@
+# Codemagic Integration
+
+[Codemagic](https://codemagic.io/) is a cloud-based CI/CD tool for Flutter apps that you can use for continuous delivery together with *fastlane*. Codemagic has *fastlane* pre-installed, so you can easily run your *fastlane* scripts as part of the Codemagic build process. 

```

Comment:
It's not specific to Flutter apps:

<img width="394" alt="image" src="https://user-images.githubusercontent.com/8419048/110227947-e6336e80-7edb-11eb-9f47-d9a3c4edc25f.png">

Reply:
Updated